lldb/test/API/lang/c/calling-conventions/TestCCallingConventions.py

```diff
@@ -6,6 +6,7 @@
 
 
 class TestCase(TestBase):
+    @skipIf(oslist=["linux"])
```

Comment:
This looks like it's dangling, it should decorate the appropriate `test_` method.